changelogs/fragments/7847-gitlab-issue-title.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - gitlab_issue - fix behavior to search gitlab issue, using `search` keyword instead of `title` (https://github.com/ansible-collections/community.general/issues/7846).
```

Comment:
```suggestion
  - gitlab_issue - fix behavior to search GitLab issue, using ``search`` keyword instead of ``title`` (https://github.com/ansible-collections/community.general/issues/7846).
```